{
    "parse_time": "2025-10-10 10:04:42",
    "path": "https://www.cnvd.org.cn/flaw/show/CNVD-2025-23279",
    "spider_time": 1760090682316,
    "name": "PyTorch存在未明漏洞（CNVD-2025-23279）",
    "cnvd": "CNVD-2025-23279",
    "publish": "2025-10-10",
    "severity": "高",
    "cvss_vector": "AV:N/AC:L/Au:N/C:N/I:N/A:C",
    "product": "[\"Pytorch Pytorch v2.7.0\"]",
    "cve": "CVE-2025-55553",
    "description": "PyTorch是PyTorch开源的一个Python包。PyTorch存在安全漏洞，该漏洞源于组件proxy_tensor.py存在语法错误，攻击者可利用该漏洞导致拒绝服务。",
    "flaw_type": "通用型漏洞",
    "reference": "https://gist.github.com/shaoyuyoung/0e7d2a586297ae9c8ed14d8706749efc",
    "solution": "厂商已发布了漏洞修复程序，请及时关注更新：\nhttps://github.com/pytorch/pytorch/releases",
    "patch": "PyTorch存在未明漏洞（CNVD-2025-23279）的补丁",
    "verify": "(暂无验证信息)",
    "submit_time": "2025-09-28",
    "open_time": "2025-10-10",
    "update_time": "2025-10-10",
    "data_summary": "ba2ab9f454498400d3954fb4b8bfb6b1"
}